.github/workflows/merge.yml

```diff
@@ -22,38 +22,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: "3. Update pip"
+      - name: "3. Tests"
         run: |
-          python -m pip install --upgrade pip
-
-      - name: "4. Attach cache for pip"
-        uses: actions/cache@v1
-        id: cache
-        with:
-          path: ~/.cache/pip
-          key: tests-${{ runner.os }}-pip-${{ hashFiles('requirements/main.txt', 'requirements/dev.txt') }}
-          restore-keys: |
-            tests-${{ runner.os }}-pip-
-
-      - name: "5-0. Install torch without cuda on ${{ runner.os }}"
-        if: contains(runner.os, 'Linux')
-        run: |
-          # this avoids an unnecessary cuda install on Linux
-          pip install -q torch==1.5.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
-
-      - name: "5. Install main and dev dependencies"
-        run: |
-          pip install -q .
-          pip install -q -r requirements/dev.txt
-
-      - name: "5. Run tests with coverage"
-        run: |
-          coverage run -m unittest
-          coverage report -m --fail-under=80
-
-      - name: "6. Run linter"
-        run: |
-          flake8 --config=setup.cfg darts
+          ./gradlew test
```

Comment:
might still need the step to eg. attach the cache for pip, probably not a gradle thing

Reply:
Yes, sorry I went into berserk delete everything mode.